server/lyft/decorators/audit_project_commands_wrapper.go

```diff
@@ -0,0 +1,140 @@
+package decorators
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/events"
+	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/lyft/aws/sns"
+)
+
+// AuditProjectCommandWrapper is a decorator that notifies sns topic
+// about the state of the command. It is used for auditing purposes
+type AuditProjectCommandWrapper struct {
+	SnsWriter sns.Writer
+	events.ProjectCommandRunner
+}
+
+func (p *AuditProjectCommandWrapper) Apply(ctx models.ProjectCommandContext) models.ProjectResult {
+	id := uuid.New()
+	startTime := time.Now()
+
+	applyEvent := &ApplyEvent{
+		Version:        1,
+		ID:             id.String(),
+		RootName:       ctx.ProjectName,
+		JobType:        models.ApplyCommand,
+		Respository:    ctx.BaseRepo.FullName,
+		Environment:    ctx.Tags["environment"],
+		PullNumber:     ctx.Pull.Num,
+		InitiatingUser: ctx.User.Username,
+		Project:        ctx.Tags["service_name"],
+		ForceApply:     ctx.ForceApply,
+		StartTime:      startTime.Format(time.RFC3339),
```

Comment:
Why not ISO? It seems like more of a standard.  RFC also has a space in it which feels weird to have when persisting things.

Reply:
`RFC3339` is ISO format at least based on googling. I took the constant based on these values https://pkg.go.dev/time#pkg-constants. There shouldn't be any space based on the format, where do you see that?

Reply:
https://stackoverflow.com/questions/522251/whats-the-difference-between-iso-8601-and-rfc-3339-date-formats

 The RFC also allows the "T" to be replaced by a space (or other character), while the standard only allows it to be omitted (and only when there is agreement between all parties using the representation).


Maybe we should just use time since epoch here to be resilient from a persistence standpoint.